_commands/basics/gcc.md

```diff
@@ -0,0 +1,106 @@
+---
+---
+
+gcc flags
+--
+
+`gcc` is a compiler command that has the ability to execute options or 'Flags' when compiling.  These flags can be grouped and executed via the command line:
+    '-o' places output in a file.
```

Comment:
For code snippets, generally prefer \`backticks\` over 'single quotes' so they `render nicely`.

Additionally, these all collapse into one ¶ when rendered:

![image](https://cloud.githubusercontent.com/assets/5882053/20015548/4173162c-a292-11e6-8ec4-7700cb909a5b.png)

Take a look at [how to format with Markdown](https://daringfireball.net/projects/markdown/).